sklearn/decomposition/dict_learning.py

```diff
@@ -962,6 +1026,7 @@ class DictionaryLearning(BaseEstimator, SparseCodingMixin):
         omp: uses orthogonal matching pursuit to estimate the sparse solution
         threshold: squashes to zero all coefficients less than alpha from
         the projection ``dictionary * X'``
+        elastic_net: use the elastic net constraint.
```

Comment:
Have updated the note to be a bit more clear. Also renamed `elastic_net` -> `elastic_net_cd`.